business/templates/jobs.html

```diff
@@ -26,11 +27,30 @@ <h1 class="section-title">Job board</h1>
                 <a class="link-unstyled mt-3 d-block" href="{% url 'business_jobs_faq' %}"><p class="mb-0"><strong>Any questions? Check out our FAQ <i class="fas fa-arrow-right ml-1"></i></strong></p></a>
             </div>
         </div>
-        {% if not offers %}
-            <hr>
-            <p class="mt-4">There's no offers at the moment but make sure to check this page soon again!</p>
+
+        <div class='col-xs-3'>
+            <a href="{% url 'business_jobs' %}" class="link-unstyle">
+                <button type="button" class="btn btn-secondary rounded mt-1 {% if type == '' %}active{% endif %}"> Show all </button>
```

Comment:
Strip spaces around "Show all".

Reply:
Probably better to `{% if not type %}`.

Reply:
Actually not because there's a type that's `0` but pretty sure can do `{% if type != None %}` here.

Reply:
Seems to be working only when the type is None, so I put `{% if not type %}`.

Reply:
Ah sorry, you want to show all when it's actually None then you need `{% if type == None %}`.